### DISCUSSION TITLE
Help needed: How to generate a high-level overview of a community's list of Pubs (including authors, categories and dois)

### DISCUSSION THREAD

**Discussion by tosteiner** [knowledgefutures/pubpub#2670] [Support] [answered]
Hi Gabe and all at PubPub,

I'm currently in the midst of writing a project report, and as we'd like to include a list of Pubs created by our community, I was wondering if there's a way to generate such a list via PubPub ( https://copim.pubpub.org )?

What I'd ideally be looking for would be a list that includes

* Pub title
* Pub author
* Date published
* Pub category / categories
* doi
* URL
* add'l usage information such as Pub visits and downloads (as per the existing Impact tab)

I'm aware that we're already able to generate lists of Pubs via the _Impact_ overview, and thanks so much for implementing that! What this is currently missing would be details on e.g. Pub authors, categories, and dois ... 

So I guess my question would be twofold:
 
a) As a short-term fix: Do you think you might be able to create a csv list with the above data for all Pubs created in copim.pubpub.org from the main dev backend? In case this helps, COPIM's PubPub community ID appears to be `785a6451-ad52-404b-bbdc-b1f16ba3da29` (as per @tefkah's [API crawler](https://pubpub.tefkah.com/) 😉  ) 
b) And with a medium-term view, and as it might also be useful for other communities: would it be possible to include more details in the Impact reporting tool to enable communities to have the above information available at their fingertips?

Thanks so much for your help with this, and all best, Toby

**Comment by gabestein** [comment, marked as answer]
Hi Toby — I'm emailing you the best version of this we can do.

Unfortunately, though, because the database is highly normalized, it's quite complex to build out. For example, we determine the time the pub was published first by looking to see if there's a custom published on date, then by checking the creation date of the last release. And that's just one field. Authors, etc., are more complex.

It's by no means impossible, so if other folks want some dashboards more like this, it would be useful to know. But it's not, unfortunately, particularly turnkey within our current analytics setup.

**Reply by tosteiner** [reply]
Hi Gabe,

oooh, that's amazing - thank you very much, this does help me a lot!

Also, no worries at all about the limitations, I figured shortly after sending my query and reading up a bit more on PubPub's documentation that what may have initially sounded as if it should be easy to pull off might in fact not be that trivial ... so thanks a million for giving it a go, and for outlining the caveats!

And indeed, perfectly understandable, of course - let's see what other users think about the idea of an extended Dashboard, and what your internal resources can actually make that happen.

Thanks as always for your timely support, really appreciate it!
